Proteomics/Protein/VariantApplication.cs

```diff
@@ -11,10 +11,9 @@ public static class VariantApplication
         /// </summary>
         /// <param name="protein"></param>
         /// <param name="sequenceVariation"></param>
-        public static string GetAccession(Protein protein, IEnumerable<SequenceVariation> appliedSequenceVariations)
+        public static string GetAccession(Protein protein,IEnumerable<SequenceVariation> appliedSequenceVariations)
         {
-                        
-            return protein.Accession + 
+            return protein.NonVariantProtein.Accession + 
```

Comment:
How does the protein's accession vary from the nonvariant protein's accession?

Reply:
in a protein with multiple variants the variants are applied one at a time. So for example the non variant protein is A and A has variants 1,2,3 and 4. When going through variant application first, variant 1 is applied and the protein. Accession becomes A_1 and the non variant protein.Accession remains A. Then variant 2 is applied to A_1. At this time all variants listed are applied. This means the protein becomes A_1_1_2 with previous code because the list of variant contains 1 and 2 and the protein. Accession is A_1. With the new code the non variant accession is what is added to since the list of variants are cumulative.